doc/.vuepress/config.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+    title: 'Gerbil Scheme',
+    description: '',
+    themeConfig: {
+        nav: [
+          { text: 'Home', link: '/' },
+          { text: 'Guide', link: '/guide/' },
+          { text: 'Tutorials', link: '/tutorials/' },
+          { text: 'StdLib', link: '/stdlib/' },
+        ],
+        sidebarDepth: 2,
+        sidebar: {
+          '/guide/': [
+            {
+              collapsable: false,
+              title: 'Guide',
+              children: ['', 'intro', 'shell', 'package-manager', 'ffi', 'build', 'profiler', 'srfi', 'core-prelude', 'r7rs', 'bootstrap', 'nix']
+            }
+          ],
+          '/stdlib/': [
+            {
+              collapsable: false,
+              title: 'Gerbil API Reference',
+              children: ['', 'actor', 'coroutine', 'crypto', 'db', 'debug', 'errors', 'events', 'format', 'generic', 'getopt', 'iterators',
+              'lazy', 'logger', 'make', 'misc', 'net', 'os', 'parser', 'regexp', 'sort', 'srfi', 'stxparam', 'sugar', 'test', 'text',
+              'web', 'xml']
+            }
+          ]
+        },
+        // Assumes GitHub. Can also be a full GitLab url.
+        repo: 'vyzo/gerbil',
+        // Customising the header label
+        // Defaults to "GitHub"/"GitLab"/"Bitbucket" depending on `themeConfig.repo`
+        repoLabel: 'Contribute!',
+
+        // Optional options for generating "Edit this page" link
+
+        // if your docs are in a different repo from your main project:
+        docsRepo: 'khepin/gerbil-docs',
```

Comment:
I think we want this inline so that we maintain it and people can easily edit the docs (hopefully we can make them great). 
I also commented on this above.